tardis/tests/test_montecarlo.py

```diff
@@ -1,9 +1,14 @@
 import numpy as np
-from tardis import montecarlo
+from tardis.montecarlo import montecarlo
+import tardis
 import pytest
 
 test_line_list = np.array([10, 9, 8, 7, 6, 5, 5, 4, 3, 2, 1]).astype(np.float64)
 
+def test_rpacket():
+    print dir(tardis.montecarlo.montecarlo)
+    assert montecarlo.rpacket_test() == True
```

Comment:
just run it. If an assertion error is being thrown pytest should already catch it.